web/src/app/common/privateRoute/PrivateRoute.js

```diff
@@ -0,0 +1,21 @@
+import { Redirect, Route } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/* global sessionStorage */
```

Comment:
is this comment actually defining a variable? I ask cause i dont see sessionStorage defined anywhere besides these comments

Reply:
Yep, exactly. Very javascript-y